src/epics_containers_cli/ioc/ioc_cli.py

```diff
@@ -64,11 +64,13 @@ def template(
         resolve_path=True,
         autocompletion=force_plain_completion,
     ),
+    debug: bool = typer.Option(False, help="Enable verbose output for helm"),
```

Comment:
I think I prefer debug true by default. This is because you would usually use this function because of a problem with the yaml. I think debug is relatively benign if the yaml is OK?